.github/workflows/ci.yaml

```diff
@@ -47,6 +47,22 @@ jobs:
             curl -L -O https://raw.githubusercontent.com/inducer/ci-support/master/run-pylint.py
             python run-pylint.py --yaml-rcfile=.pylintrc.yml "$(basename $GITHUB_REPOSITORY)" test examples && echo "Pylint found no errors."
 
+    pydocstyle:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
+        -
+          uses: actions/setup-python@v1
+          with:
+            python-version: '3.x'
+        - name: Lint
+          run: |
+            python3 -m venv myenv
+            source myenv/bin/activate
+            python -m pip install wheel
+            python -m pip install pydocstyle
```

Comment:
we could consider removing this ci section and absorb it above with `flake8-docstrings` (and possibly other flake8 extensions.